components/Common/ThemeToggle/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { MoonIcon, SunIcon } from '@heroicons/react/24/outline';
+import { AccessibleIcon } from '@radix-ui/react-accessible-icon';
+import { useTheme } from 'next-themes';
+import type { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import styles from './index.module.css';
+
+const ThemeToggle: FC = () => {
+  const intl = useIntl();
+  const { theme, setTheme } = useTheme();
+
+  const isDark = theme === 'dark';
+
+  const toggleTheme = (isKeyPress?: boolean) => {
+    if (isKeyPress) {
+      return;
+    }
+
+    setTheme(isDark ? 'light' : 'dark');
+  };
+
+  return (
+    <button
+      type="button"
+      onClick={() => toggleTheme()}
+      onKeyDown={() => toggleTheme(true)}
```

Comment:
Why do you have a function that does nothing if it's key press? What is the point here?